lib/tfpdf/wp-tfpdf.php

```diff
@@ -0,0 +1,134 @@
+<?php
+/**
+ * To avoid modifying tFPDF directly we introduce WP_tFPDF, 
+ * A thin wrapper around the parent class to utilize WP_Filesystem 
+ * 
+ * Parent's coding standards are preserved deliberately.
+ */
+// phpcs:disable WordPress.WP.AlternativeFunctions.file_system_read_fwrite, WordPress.WP.AlternativeFunctions.file_system_read_fread, WordPress.WP.AlternativeFunctions.file_system_read_fopen, WordPress.WP.AlternativeFunctions.file_system_read_fclose, WordPress.VIP.FileSystemWritesDisallow
+if ( ! defined( 'ABSPATH' ) ) exit; // Exit if accessed directly
+
+require_once ABSPATH . '/wp-admin/includes/file.php';
+
+class WP_tFPDF extends tFPDF {
+	function __construct( $orientation = 'P', $unit = 'mm', $size = 'A4' ) {
+		parent::__construct( $orientation, $unit, $size );
+		$this->init_wp_filesystem();
+	}
+	
+	function init_wp_filesystem() {
+		global $wp_filesystem;
+
+		if ( ! is_a( $wp_filesystem, 'WP_Filesystem_Base' ) ) {
+			$creds = request_filesystem_credentials( site_url() );
+			$fs = wp_filesystem( $creds );
+			return $fs ?: new WP_Error( 'fs-init-error', "Couldn't initialize Filesystem" );
+		}
+		
+		return true;
+	}
+	
+	function Output($name='', $dest='')
+{	
```

Comment:
Since this is code that we're going to be maintaining in the plugin, we should make sure it matches our coding standards (indentation, spacing around parentheses, etc).

See the comment below on how we may be able to reduce the amount of code here.